server/app/views/BaseHtmlView.java

```diff
@@ -114,38 +112,6 @@ protected ContainerTag renderPaginationDiv(
     return div.with(br());
   }
 
-  protected ContainerTag renderSearchForm(
```

Comment:
Note to the reviewer: There was only one other callsite of this and the contained logic wasn't super interesting / general. I preemptively removed it since I wanted to add more controls to the form (time filters) and it didn't currently support it.

Rather than trying to make this more general, I thought it better to remove the layer of indirection.